Tests/APITesters/ObjCAPITester/ObjCAPITester/RCConfigurationAPI.m

```diff
@@ -14,7 +14,7 @@ @implementation RCConfigurationAPI
 + (void)checkAPI {
     RCConfigurationBuilder *builder = [RCConfiguration builderWithAPIKey:@""];
     RCConfiguration *config __unused = [[[[[[[[[[[[builder withApiKey:@""]
-                                                 withObserverMode:false]
+                                                 withObserverMode:false storeKitVersion: RCStoreKitVersionStoreKit2]
```

Comment:
Objective-C doesn't get a version with the default parameter by default unless we make it an override, but I guess this is fine?
```suggestion
                                                 withObserverMode:false storeKitVersion:RCStoreKitVersionStoreKit2]
```